language/move-lang/README.md

```diff
@@ -17,15 +17,44 @@ Move source language is an ergonomic language for writing Modules and Scripts th
 Move source language is an expression-based language that aims to simplify writing Move programs---modules and scripts---without hiding the core concepts in Move bytecode.
 
 Currently, there are command line tools for Move.
+
 * Move Check is used for checking code, but it does not generate bytecode
 * Move Build is used for checking and then compiling to bytecode
+
 In the future there should be other utilities for testing and play grounding the Move modules.
 
 There is unfortunately no documentation for the language syntax or features. See the stdlib for examples.
 
+## Design Principles
+
+### Mission
+
+Deliver a minimalistic, expressive, safe, and transparent language to produce--and link with--Move bytecode.
+
+### Primary Principles
+
+* **More Concise than Bytecode** Move is expression based, which lends itself to concise and composed programs without the need for extra locals or structure. In some sense, expressions allow for programming directly on the stack in a controlled environment, and in that way, the language gives the same level of functionality in a more concise and readable environment.
```

Comment:
"programming directly on the stack in a controlled environment..." -> with a controlled and safe model? not sure about "environment".
"the language gives the same level of functionality in a more concise.." same level of functionality than what? it's not clear to me, direct bytecode programming?

Reply:
It might also be good to explicitly state that the bytecode is stack based, for the sake of someone reading this who isn't already familiar with Move bytecode. Otherwise, "programming directly on the stack" is confusing, since people may be thinking of a function call stack.